### ISSUE TITLE
Storage reporting gets confused when cluster has only half-enabled file/sharedfile storage

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#570]
Originally reported of Google Code with ID 514.

```
<b>What steps will reproduce the problem?</b>
1. Setup a cluster with file/sharedfile storage being enabled, but the path/srv/ganeti/file-storage not existing (this is currently possible)  
2. execute gnt-instance add -I ... for a drbd instance

Error message: Can't get data for node NODE: Error while executing backend function: Failed to retrieve 
                file system information about path: /srv/ganeti/file-storage - No such file or directory

Possible solutions:
- make iallocator only call for storage info for the instance to be allocated
- make sure enabled-disk-templates and the autoconf enableing of file/shared file are consisten
- make sure that file/sharedfile storage is only enabled when the directory is actually existing
```
Originally added on 2013-07-03 16:27:52 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed with commit 2fe1e0437d650c2fd9da625fd076e51336918c7e
Author: Helga Velroyen <helgav@google.com>
Date:   Tue Jul 23 15:54:51 2013 +0200

    Remove obsolete autoconf variable from remaining files
    
    This removes the obsolete autoconf variable
    'ENABLE_SHARED_FILE_STORAGE' from all remaining files.
    
    Signed-off-by: Helga Velroyen <helgav@google.com>
    Reviewed-by: Thomas Thrainer <thomasth@google.com>

```
Originally added on 2013-09-26 15:03:38 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Ganeti 2.9.0 has been released
```
Originally added on 2013-11-05 13:56:36 +0000 UTC.

Changed State: Released
